akka-distributed-data/src/main/scala/akka/cluster/ddata/PNCounterMap.scala

```diff
@@ -72,10 +72,11 @@ final class PNCounterMap[A] private[akka] (
   def size: Int = underlying.size
 
   /**
-   * Increment the counter by 1.
+   * Increment the counter with the delta specified.
+   * If the delta is negative then it will decrement instead of increment.
    */
-  def increment(key: A)(implicit node: SelfUniqueAddress): PNCounterMap[A] =
-    increment(node, key, 1)
+  def incrementBy(key: A, delta: Long = 1)(implicit node: SelfUniqueAddress): PNCounterMap[A] =
```

Comment:
shouldn't we remove the default value `= 1`?

Reply:
Preferable, yes; thought it was still wanted but I'd prefer to remove.

Reply:
I think we removed from PNCounter so then we should be consistent and remove it here too. I agree that we should remove the default. If someone complains loudly we could add it later without breaking compatibility by adding an overload.